<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="EPINHYDW0086" name="com.atm.modulefive.webdriver.advanced.tests.VoloTeaTest2" tests="2" failures="1" timestamp="15 Jun 2017 11:25:40 GMT" time="38.152" errors="0">
  <testcase name="FlightInformation" time="10.073" classname="com.atm.modulefive.webdriver.advanced.tests.VoloTeaTest2">
    <failure type="java.lang.AssertionError" message="Exception to wait for page load or Timeout Exception">
      <![CDATA[java.lang.AssertionError: Exception to wait for page load or Timeout Exception
	at org.testng.Assert.fail(Assert.java:93)
	at com.atm.modulefive.webdriver.advanced.utils.ActionUtility.waitForPageLoaded(ActionUtility.java:43)
	at com.atm.modulefive.webdriver.advanced.pageobjects.VoloTeaFlightSummary.isFlightDisplayed(VoloTeaFlightSummary.java:35)
	at com.atm.modulefive.webdriver.advanced.tests.VoloTeaTest2.FlightInformation(VoloTeaTest2.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- FlightInformation -->
  <testcase name="SearchFlights" time="28.079" classname="com.atm.modulefive.webdriver.advanced.tests.VoloTeaTest2"/>
</testsuite> <!-- com.atm.modulefive.webdriver.advanced.tests.VoloTeaTest2 -->
